.dockerignore

```diff
@@ -4,3 +4,6 @@ integration/localnet/data/
 integration/localnet/trie/
 integration/localnet/docker-compose.nodes.yml
 integration/localnet/targets.nodes.json
+
+go.work
+go.work.sum
```

Comment:
why not committing them?

Reply:
go.work structure is the same as the go.mod, but the whole idea of a separate file is so that it is not accidentally committed to the repo, so that I would assume there won't ever be need for it in git: https://go.googlesource.com/proposal/+/master/design/45713-workspace.md

For docker, it is a bit trickier, since most of the overrides use local paths they would break the Docker build (esp. w/ relic) so we should likely ignore them in docker too.

Reply:
Oh I see. It's meant to replace the `replace directive` which is mostly for local development